### DISCUSSION TITLE
1ra. practica

### DISCUSSION THREAD

**Discussion by EduGarcia502** [Launch-X-Latam/MisionFrontEnd#47] [Q&A]
hola un gusto saludarlos explorers, tengo una duda la practica del intro para cuando es, ya me ley toda la intro de la misión frontEnd, aun no entiendo si lo aremos durante las clases en vivo o como? disculpen mi duda... saludos desde Guatemala

**Comment by EricYB** [comment]


**Reply by EduGarcia502** [reply]
en que tienes problemas bro?

**Comment by JhonFre7** [comment]
Toca esperar a mañana para que resuelvan todas las dudas en el live

**Comment by doguedogue** [comment]
Lo van ir indicando conforme los vayamos haciendo.

Podría ser por ejemplo que en una semana veamos un tema, y tienes el fin de semana para hacer la práctica.

**Comment by Amhedriel** [comment]
Buenas EduGarcía, hasta donde tengo entendido el Miércoles a las 19:00 -5 de México se hablarán algunos temas relacionados con la primera misión, responderá dudas que estemos colocando en las discusiones y nos guiará en lo necesario, sin embargo, nos exhorta a buscar información más allá de lo escrito en esta plataforma, ya sea como ejemplo: Git y GitHub, para poder avanzar algo más en futuros temas, en lo personal recomiendo que aprendas algo de Mark Down que es muy útil en estos proyectos de GitHub y FrontEnd, podrás ver una guía si haces click en el símbolo de una "m" con flecha hacia abajo, que se encuentra a la derecha, abajo en estos recuadros al escribir un mensaje, saludos.

**Reply by AbateSalvador** [reply]
Si, suele ser la recomendación y dinámica que usa el mission commander Rodrigo. Yo les sugiero esperar a la sesión del miércoles.
